spec/controllers/api/sequences/sequences_create_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+describe Api::SequencesController do
+
+  include Devise::TestHelpers
+
+  describe '#create' do
+
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'creates a new sequences for a user' do
+      sign_in user
+      inputs = {name: "Scare Birds",
+                steps: [{
+                    message_type: 'move_rel'
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.